packages/typespec-test/test/openai/generated/typespec-ts/package.json

```diff
@@ -1,7 +1,5 @@
 {
   "name": "@msinternal/openai",
-  "sdk-type": "client",
```

Comment:
I think we probably should keep the sdk-type here? I remember this is the configuration for the pipeline ?

Reply:
This is only relevant for azureSdkForJs === true